build/xtask/src/dist.rs

```diff
@@ -1213,6 +1217,9 @@ fn update_image_header(
                 // Alias for the NS peripherals
                 sau_ranges.insert(0x4000_0000..=0x4fff_ffff, false);
 
+                // Alias for NS SRAMs
+                sau_ranges.insert(0x2002_0000..=0x2004_ffff, false);
```

Comment:
Is this change intentional?

Reply:
Yes, otherwise you can't use `sram2` for a dump area. (It's being used in the LPCXpresso image.)

Reply:
👍🏻